setup.py

```diff
@@ -0,0 +1,23 @@
+from setuptools import setup
+
+setup(
+    name='tensorizer',
+    version='0.0.1',
+    description='A tool for PyTorch Module, Model, and Tensor Serialization/Deserialization.',
+    url='https://github.com/coreweave/tensorizer',
+    author='CoreWeave',
+    license='MIT',
+    packages=['tensorizer'],
+    install_requires=[
+        'torch',
+        'protobuf>=3.19.5',
```

Comment:
I know there have been issues with newer protobuf before, perhaps we should limit this to this major version?

Reply:
Maybe if we regenerated stubs as part of the install? Not sure if that's what we want to do.